app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+
+import classnames from "classnames";
+import { Plus, MenuHorizontal } from "neetoicons";
+import { Button, Table, Dropdown } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+import { useTranslation } from "react-i18next";
+import { noop } from "utils";
+
+import MenuBar from "components/Dashboard/MenuBar";
+import { SINGULAR, PLURAL } from "constants";
+
+import {
+  COLUMN_DATA,
+  ROW_DATA,
+  COMMON_MENU_BLOCKS,
+  DEFAULT_PAGE_SIZE,
+} from "./constants";
+
+const Contacts = () => {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+  const [selectedRowIds, setSelectedRowIds] = useState([]);
+
+  const {
+    Menu,
+    MenuItem: { Button: MenuButton },
+  } = Dropdown;
+
+  const { t } = useTranslation();
+
+  const handleRowSelect = ids => setSelectedRowIds(ids);
+
+  const columnData = COLUMN_DATA.map(column => {
+    if (column.key === "action") {
+      return {
+        ...column,
+        render: () => (
+          <Dropdown buttonStyle="text" className="p-4" icon={MenuHorizontal}>
+            <Menu>
+              <MenuButton onClick={noop}>{t("actions.edit")}</MenuButton>
+              <MenuButton onClick={noop}>{t("actions.delete")}</MenuButton>
+            </Menu>
+          </Dropdown>
+        ),
+      };
+    }
+
+    return column;
+  });
```

Comment:
You can move this to a `buildColumnData` utils function.

Reply:
This was taken care of in #15